src/apis/vaults.ts

```diff
@@ -36,9 +39,16 @@ export class DefaultVaultsAPI {
         return vaultsMap.map((v) => v[1]);
     }
 
-    async mapForVault(vaultId: AccountId): Promise<Map<AccountId, IssueRequest[]>> {
-        // cannot instantiate issueAPI in the constructor, because that would create a dependency loop,
-        // since issueAPI also instantiates the vaultsAPI in its constructor
+    async mapIssueRequests(vaultId: AccountId): Promise<Map<AccountId, IssueRequest[]>> {
+        /**
```

Comment:
Nice job adopting the new comment format. Two nitpicks:
- doc comments should be above the function definition
- the first line of the doc comment should be a brief description of the function